runtime/doc/options.txt

```diff
@@ -2461,6 +2461,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	"stlnc" the space will be used when there is highlighting, '^' or '='
 	otherwise.
 
+        Note that "horiz", "horizup", "horizdown", "vertleft", "vertright" and
+        "verthoriz" are only used when 'laststatus' is 3, since only vertical
+        window separators are used otherwise.
```

Comment:
Tabs were used for indent below and above this, so perhaps it is better to use tabs here.
```suggestion
	Note that "horiz", "horizup", "horizdown", "vertleft", "vertright" and
	"verthoriz" are only used when 'laststatus' is 3, since only vertical
	window separators are used otherwise.
```

Reply:
Should be fixed now